x-pack/test/saved_object_api_integration/spaces_only/apis/bulk_update.ts

```diff
@@ -16,31 +16,50 @@ const {
 } = SPACES;
 const { fail404 } = testCaseFailures;
 
-const createTestCases = (spaceId: string) => [
+const createTestCases = (spaceId: string) => {
   // for each outcome, if failure !== undefined then we expect to receive
   // an error; otherwise, we expect to receive a success result
-  { ...CASES.SINGLE_NAMESPACE_DEFAULT_SPACE, ...fail404(spaceId !== DEFAULT_SPACE_ID) },
-  { ...CASES.SINGLE_NAMESPACE_SPACE_1, ...fail404(spaceId !== SPACE_1_ID) },
-  { ...CASES.SINGLE_NAMESPACE_SPACE_2, ...fail404(spaceId !== SPACE_2_ID) },
-  {
-    ...CASES.MULTI_NAMESPACE_DEFAULT_AND_SPACE_1,
-    ...fail404(spaceId !== DEFAULT_SPACE_ID && spaceId !== SPACE_1_ID),
-  },
-  { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_1, ...fail404(spaceId !== SPACE_1_ID) },
-  { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_2, ...fail404(spaceId !== SPACE_2_ID) },
-  CASES.NAMESPACE_AGNOSTIC,
-  { ...CASES.HIDDEN, ...fail404() },
-  { ...CASES.DOES_NOT_EXIST, ...fail404() },
-];
+  const normal = [
+    { ...CASES.SINGLE_NAMESPACE_DEFAULT_SPACE, ...fail404(spaceId !== DEFAULT_SPACE_ID) },
+    { ...CASES.SINGLE_NAMESPACE_SPACE_1, ...fail404(spaceId !== SPACE_1_ID) },
+    { ...CASES.SINGLE_NAMESPACE_SPACE_2, ...fail404(spaceId !== SPACE_2_ID) },
+    {
+      ...CASES.MULTI_NAMESPACE_DEFAULT_AND_SPACE_1,
+      ...fail404(spaceId !== DEFAULT_SPACE_ID && spaceId !== SPACE_1_ID),
+    },
+    { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_1, ...fail404(spaceId !== SPACE_1_ID) },
+    { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_2, ...fail404(spaceId !== SPACE_2_ID) },
+    CASES.NAMESPACE_AGNOSTIC,
+    { ...CASES.HIDDEN, ...fail404() },
+    { ...CASES.DOES_NOT_EXIST, ...fail404() },
+  ];
+
+  // an "object namespace" string can be specified for individual objects (to bulkUpdate across namespaces)
+  // even if the Spaces plugin is disabled, this should work, as `namespace` is handled by the Core API
```

Comment:
nit: I don't think this comment makes sense in the `spaces_only` test suite, since spaces would always be enabled

Reply:
Good catch, this was sloppy copy-pasting on my part!